las/prediction.py

```diff
@@ -21,11 +21,11 @@ def confidence(self):
 
 
 class Prediction(Dict[str, str]):
```

Comment:
I know this is old code, but inheriting from dict doesn't make any sense, prediction doesn't have anything to do with it.

Reply:
Same applies to `Field`.